aws_lambda_powertools/utilities/parameters/base.py

```diff
@@ -92,12 +94,12 @@ def get(
         except Exception as exc:
             raise GetParameterError(str(exc))
 
-        if transform is not None:
+        if transform:
             if isinstance(value, bytes):
                 value = value.decode("utf-8")
             value = transform_value(value, transform)
-
-        self.store[key] = ExpirableValue(value, datetime.now() + timedelta(seconds=max_age))
+            if value:
+                self.store[key] = ExpirableValue(value, datetime.now() + timedelta(seconds=max_age))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)

Reply:
I don't think it is relevant here. We use naive datetime object but still we compare those dates in the same timezone (region) so there is no risk of comparing two diferent datetime object with different timezones leading to wrong ttl's values.
@heitorlessa Please advise if you see any other problems here and what is your recommandation.

Reply:
What!?!? I didn't post this HAHA. ignore it.